RecoTracker/Configuration/python/customiseForRunI.py

```diff
@@ -109,4 +109,23 @@ def customiseForRunI(process):
     process.SiStripClusterChargeCutTight.value = -1.
     process.SiStripClusterChargeCutLoose.value = -1.
 
+
+    if 'TrackSeedMonjetCoreRegionalStep' in process.TrackingDQMSourceTier0Common.moduleNames():
+      process.TrackingDQMSourceTier0Common.remove(process.TrackSeedMonjetCoreRegionalStep)
+
+    if 'TrackSeedMonjetCoreRegionalStep' in process.TrackingDQMSourceTier0.moduleNames():
+       process.TrackingDQMSourceTier0.remove(process.TrackSeedMonjetCoreRegionalStep)
+
+    if 'TrackSeedMonjetCoreRegionalStep' in process.TrackingDQMSourceTier0MinBias.moduleNames():
+       process.TrackingDQMSourceTier0MinBias.remove(process.TrackSeedMonjetCoreRegionalStep)
+
+    if hasattr(process, 'TrackingDQMSourceTier0Common') and 'TrackSeedMonjetCoreRegionalStep' in process.TrackingDQMSourceTier0Common.moduleNames():
```

Comment:
C'mon guys!
This is plain wrong!
Please remove lines from 113:121 out this PR.
Sorry if I was not clear enough on my (private) patch exchange....